cmd/kinflate/Makefile

```diff
@@ -0,0 +1,28 @@
+SHELL := /bin/bash -euo pipefail
+
+config_file_name = Kube-manifest.yaml
+example_config = docs/$(config_file_name)
+
+.PHONY: all
+all: docs
+
+# In a branch, run 'make docs' to update docs with
+# generated code, then merge it to master.
+docs: $(example_config)
+
+# Use kinflate to create the standard kinflate configuration
+# file that appears in the website's documentation.
+$(example_config): /tmp/bin/kinflate
+	rm -f TMP
+	echo "################################" > TMP
+	echo "# GENERATED FILE - do not edit." >> TMP
+	echo "# To generate:  make docs" >> TMP
+	echo "################################" >> TMP
+	echo " " >> TMP
+	/tmp/bin/kinflate init
+	cat $(config_file_name) >> TMP
+	mv TMP $(example_config)
+	rm $(config_file_name)
+
+/tmp/bin/kinflate:
+	GOPATH=/tmp go get k8s.io/kubectl/cmd/kinflate
```

Comment:
```
go build -o /tmp/bin/kinflate kinflate.go
```
Using above command to build temporary binary will be safer here because it works with current checked out repo as is. If developer has checked out the repo in path other than `k8s.io/kubectl`, current approach will fetch the repo in `k8s.io...` instead of using current repo.